projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1978,7 +1978,7 @@ public function process_submission() {
 		$dashboard_url           = '';
 		$footer_mark_as_spam_url = '';
 		if ( $feedback_status !== 'jp-temp-feedback' ) {
-			$dashboard_url           = ( new Dashboard_View_Switch() )->get_forms_admin_url( $status, true ) . '&r=' . $post_id;
+			$dashboard_url           = Forms_Dashboard::get_forms_admin_url( $status ) . '&r=' . $post_id;
```

Comment:
Nitpick: $status is a confusing name, $tab would work better.

Reply:
I'll agree with you semantically, but as the tab also represents the post status, I think it can stay. Besides, it's unrelated to the main goal of this PR.